Makefile

```diff
@@ -91,11 +91,12 @@ TARGETS += vmtoolsd-guest-agent
 TARGETS += wasmedge
 TARGETS += xen-guest-agent
 TARGETS += zfs
+TARGETS += dvb-si2168-firmware
 NONFREE_TARGETS = nonfree-kmod-nvidia
 
 # help menu
 
-export define HELP_MENU_HEADER
+define HELP_MENU_HEADER
```

Comment:
My make would otherwise error out. I don't know if I'm the only one that has that issue with the way it was defined previously.